relay-event-normalization/src/event.rs

```diff
@@ -2592,23 +2598,18 @@ mod tests {
     }
 
     #[test]
-    fn test_user_data_moved() {
-        let mut user = Annotated::new(User {
-            other: {
-                let mut map = Object::new();
-                map.insert(
-                    "other".to_string(),
-                    Annotated::new(Value::String("value".to_owned())),
-                );
-                map
-            },
-            ..User::default()
-        });
-
-        normalize_user(&mut user);
-
-        let user = user.value().unwrap();
+    fn test_normalize_user() {
+        let json = r#"{
+            "user": {
+                "id": "123456",
+                "username": "john",
```

Comment:
Added `username` to validate the priority of fields given in the `get_eventuser_tag` function used in `normalize_user`.